### ISSUE TITLE
Fix Test Set doc to include datamodule

### ISSUE THREAD

**Issue by drozzy** [PyTorchLightning/pytorch-lightning#5265]
In the test set documentation it still uses the old `test_dataloader` format and not the new datamodule argument:
https://pytorch-lightning.readthedocs.io/en/stable/test_set.html

**Comment by rohitgr7** [comment]
the heading says `Test with additional data loaders`, so I believe it's correct. Maybe add a section for `Test with data modules`??

**Comment by drozzy** [comment]
I think I saw somewhere that `train_dataloader` approach was no longer recommended?

**Comment by rohitgr7** [comment]
It's just another option but both dataloaders and datamodules are supported.

**Comment by drozzy** [comment]
Ah ok.
